src/Compilers/CSharp/Portable/Symbols/Synthesized/GeneratedLabelSymbol.cs

```diff
@@ -9,9 +9,19 @@ namespace Microsoft.CodeAnalysis.CSharp.Symbols
 {
     internal sealed class GeneratedLabelSymbol : LabelSymbol
     {
+        private readonly string _name;
```

Comment:
I searched for this pattern in our codebase and found 80 occurences. Since there is nothing wrong with it technically, I will leave it as is for consistency.